### PR TITLE
Rework der Seite

### PR DESCRIPTION
**Pull request by BLACK4585**
Changed:

- added background
- moved all elements
- redesigned all elements
- addded statusbar
- improved accessibility for screenreaders eg
- changed the system missing inputs are displayed in the status bar
- added footer with links to this github and tomatenkuchen discord
- added headline
- added predefined dates for the short url
- changed the min value from custom date to current time + two minutes for more safety
- default date is 1 day

Thats it, todos in readme will continue to work on them

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,69 +6,53 @@
...
+			<fieldset>
+				<legend>Status:</legend>
+				<h3 id="response">Warte auf Eingabe...</h3>
+			</fieldset>
+		</main>
+		<footer>
+			<a href="https://github.com/devtomatocake/shorter">
+				<img src="./assets/gh.png" alt="GitHub" width="32" height="32">
+			</a>
+			<a href="https://discord.gg/ZqzFUC8qe9">
```
Comment (DEVTomatoCake) [inline index.html:54]: Closing </a> tag is missing here

File: index.html [hunk 2]
```diff
@@ -6,69 +6,53 @@
...
+					<option value="1d" selected>1 Tag</option>
+					<option value="3d">3 Tage</option>
+					<option value="1w">1 Woche</option>
+					<option value="2w">2 Wochen</option>
+					<option value="1mo">1 Monat</option>
+					<option value="2mo">2 Monate</option>
+					<option value="3mo">3 Monate</option>
+					<option value="6mo">6 Monate</option>
+					<option value="1y">1 Jahr</option>
+					<option value="userdef">Benutzerdefiniert</option>
```
Comment (DEVTomatoCake) [inline index.html:37]:
```suggestion
					<option value="custom">Benutzerdefiniert</option>
```

File: index.html [hunk 3]
```diff
@@ -6,69 +6,53 @@
...
-			<br>
-			<button type="submit">Erstellen</button>
-		</form>
+			<fieldset>
+				<legend>Status:</legend>
+				<h3 id="response">Warte auf Eingabe...</h3>
+			</fieldset>
+		</main>
+		<footer>
+			<a href="https://github.com/devtomatocake/shorter">
```
Comment (DEVTomatoCake) [inline index.html:51]:
```suggestion
			<a href="https://github.com/DEVTomatoCake/Shorter">
```

File: index.js [hunk 4]
```diff
@@ -0,0 +1,99 @@
+/* eslint-disable no-unused-vars */
```
Comment (DEVTomatoCake) [inline index.js:1]: imagine eslint

File: index.js [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+            dateElem.min = dateAdd(new Date, "minute", 2).toISOString().slice(0, -8)
+            dateElem.value = dateAdd(new Date, "minute", 2).toISOString().slice(0, -8)
+            break;
+        default: document.getElementById("date_userdef").type = "hidden"; 
+            break;
+    }
+}
+
+function dateAdd(date, interval, units) {
+    if(!(date instanceof Date))
```
Comment (DEVTomatoCake) [inline index.js:55]:
```suggestion
    if (!(date instanceof Date))
```

File: index.js [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+      case 'month'  :  ret.setMonth(ret.getMonth() + units); checkRollover();  break;
+      case 'week'   :  ret.setDate(ret.getDate() + 7*units);  break;
+      case 'day'    :  ret.setDate(ret.getDate() + units);  break;
+      case 'hour'   :  ret.setTime(ret.getTime() + units*3600000);  break;
+      case 'minute' :  ret.setTime(ret.getTime() + units*60000);  break;
+      case 'second' :  ret.setTime(ret.getTime() + units*1000);  break;
+      default       :  ret = undefined;  break;
+    }
+    return ret;
+}
```
Comment (DEVTomatoCake) [inline index.js:71]: "Sag mir, dass du Code ausm Internet kopiert hast ohne mir zu sagen, dass du Code ausm Internet kopiert hast"

File: index.js [hunk 7]
```diff
@@ -0,0 +1,99 @@
...
+    let select = document.getElementById('date');
+    let option = select.options[select.selectedIndex];
+    switch (option.value) {
+        case "userdef": return document.getElementById("date_userdef").value;
+        default: return toDate(option.value).toISOString().slice(0, -8);
+    }
+}
+
+function toDate(value){
+    let select = document.getElementById('date');
```
Comment (DEVTomatoCake) [inline index.js:83]:
```suggestion
    let select = document.getElementById("date");
```

File: style.css [hunk 8]
```diff
@@ -0,0 +1,135 @@
...
+button:active {
+    background: #2b38cf;
+    border: 3px solid #0077ff;
+    box-shadow: 0 0 50px #0b487e;
+    border-radius: 5px;
+    margin: 7px 0;
+    padding: 5px;
+    font-family: Helvetica, sans-serif;
+    font-size: 130%;
+}
```
Comment (DEVTomatoCake) [inline style.css:115]: Remove unchanged values here or group them together in a new selector if needed

File: index.js (hunk 5 above)
Comment (DEVTomatoCake) [reply index.js:55]: Applies to all function names and if's, so `function update(){` -> `function update() {`

**Review by DEVTomatoCake — CHANGES_REQUESTED** [review]
CODÄÄÄÄÄÄÄÄÄN

File: index.js (hunk 4 above)
Comment (BLACK4585) [reply index.js:1]: imagine nicht :feelslookingman:

File: index.js [hunk 9]
```diff
@@ -0,0 +1,99 @@
...
+      case 'hour'   :  ret.setTime(ret.getTime() + units*3600000);  break;
+      case 'minute' :  ret.setTime(ret.getTime() + units*60000);  break;
+      case 'second' :  ret.setTime(ret.getTime() + units*1000);  break;
+      default       :  ret = undefined;  break;
+    }
+    return ret;
+}
+
+function getDate(){
+    let select = document.getElementById('date');
```
[the remark this answers is not in the document]
Comment (BLACK4585) [reply index.js:74]: *hust* schau mal bei dir in setup()

**Review by DEVTomatoCake — APPROVED** [review]
:hugBlack:

**Pull request merged**